scripts/bootstrap/buildbootstrapcli.sh

```diff
@@ -156,7 +156,7 @@ fi
 __build_arch=${__runtime_id#*-}
 
 if [[ -z "$__outputpath" ]]; then
-   __outputpath=`getrealpath $__runtime_id/dotnetcli`
+   __outputpath=$__runtime_id/dotnetcli
```

Comment:
Should the `getrealpath` should be removed from the arg block, too, for the same reason?

https://github.com/dotnet/source-build/blob/c80149fc77de987d0270ded05c7f066f1589a114/scripts/bootstrap/buildbootstrapcli.sh#L135-L138

Reply:
I wasn't sure what the design intention here was. I will make this change.

Reply:
Done.